### ISSUE TITLE
Usability of Facet Links

### ISSUE THREAD

**Issue by greebie** [archivesunleashed/warclight#43]
I have started using Web Archives for some paper research and I was not quick to realize that the "+" was the appropriate link for setting a filter.

This was for a search for "prorogue" with a desired filter for 2008.

I wanted to click on the date itself and/or the number of entries. 

I realize that this is probably a BlackLight thing, but it might be worth considering the alteration in the case of Web Archives because dates, formats etc. aren't as thick with meaning as other examples I've seen around (e.g. subject headings etc.).

**Comment by ruebot** [comment]
Can you provide a screenshot with what you're referring to marked? I'm not entirely sure what you're referring to.

**Comment by greebie** [comment]
(This is at webarchives.ca btw.)

![image](https://user-images.githubusercontent.com/111091/39480503-73496d9a-4d36-11e8-97ef-1b503e9063b6.png)

In this above case, if I want a filter, I need to click the "+" (currently highlighted in green) to reduce the number of pages in the search.  My instinct was to click on the number 222.  I also tried the 2008 before I eventually landed on the +.

I may just be weird on this too, but I thought I'd share it, while I noticed it.

Potential solutions: make the 222 seem more like text and less like a button.  Maybe either kill the blue colour for the 2008 or make it clickable like the plus.
